pkgs/development/tools/vsce/generate.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p node2nix
+
+VSCE_ROOT="$(
+    cd "$(dirname "$0")"
+    pwd
+)"
+pushd $VSCE_ROOT 1>/dev/null
```

Comment:
Line 4 to 8 can be replaced with this:

```suggestion
cd "$(dirname "$0")"
```